.circleci/config.yml

```diff
@@ -175,7 +175,7 @@ jobs:
            command: tar -xzf /tmp/workspace/linux_amd64/please_*.tar.gz
        - run:
            name: Cross-compile
-           command: ./please/please build -p --profile ci --arch freebsd_amd64 //package:release_files
+           command: ./please/please build -p --profile ci --arch freebsd_amd64 -o please.location:~/please/please //package:release_files
```

Comment:
Hmmm, curious why these need to change?

Reply:
I've set the please location config value to `./plz-out/please` in `.plzconfig` to aid development. At the moment we have to set the location to the full path in `.plzconfig.local` which is a bit annoying.